x-pack/plugins/observability_solution/apm/public/components/app/mobile/service_overview/geo_map/embedded_map.test.tsx

```diff
@@ -35,6 +35,12 @@ describe('Embedded Map', () => {
       metaFields: [],
     } as unknown as DataView;
 
+    const styleColors = {
```

Comment:
This seems like some kind of workaround, is it really necessary?
Could we set these values as defaults if the prop is not provided?